src/components/search-form.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+
+import UserInfo from '../stores/user-info';
+import UserAction from '../actions/user-action';
+
+export default class SearchForm extends React.Component {
```

Comment:
Now it's quite hard to understand where is each form.
I clicked on `Add new user` several times instead of `Find` button

Currently we have two 'merged' forms 
![image](https://user-images.githubusercontent.com/12034551/96554832-b9c9dd80-12bf-11eb-9326-b8c09645a310.png)

They should be visually separated, e.g. Searching area and `Add new user` form
![image](https://user-images.githubusercontent.com/12034551/96554886-cb12ea00-12bf-11eb-8989-a626db819912.png)